### PR TITLE
Fix taiwind config

### PR DESCRIPTION
**Pull request by tanphat195**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,9 +25,9 @@
   "scripts": {
     "build:css": "postcss src/index.css -0 src/tailwind.css",
     "watch:css": "post src/index.css -0 src/tailwind.css -w",
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
+    "start": "craco start",
+    "build": "craco build",
+    "test": "craco test",
```
Comment (TanTai55) [inline package.json:30]: Why using craco instead of react-script

File: tailwind.config.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+  purge: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
+  theme: {
+    extend: {
+      backgroundColor: {
+        'black-t-50': 'rgba(0,0,0,0.5)',
+      }
+    }
+  },
+  variants: {},
+  plugins: []
```
Comment (TanTai55) [inline tailwind.config.js:13]: Thank you

File: package.json (hunk 1 above)
Comment (tanphat195) [reply package.json:30]: Because i use this by document https://tailwindcss.com/docs/guides/create-react-app

**Review by TanTai55 — APPROVED** [review]
LGTM!

**Pull request merged**